server/controllers/plans-controller.js

```diff
@@ -30,7 +31,7 @@ const plans = (req, res) => {
             .send(response.data);
         })
         .catch((error) => {
-          console.log(error);
+          Logger.log(error);
```

Comment:
pay attention, there's another call to `Logger.error(error);` just below. Maybe we can remove this line.

Reply:
Removed